storage-resize-images/functions/src/config.ts

```diff
@@ -19,4 +19,5 @@ export default {
   cacheControlHeader: process.env.CACHE_CONTROL_HEADER,
   imageSizes: process.env.IMG_SIZES.split(","),
   resizedImagesPath: process.env.RESIZED_IMAGES_PATH,
+  deleteOriginalFile: process.env.DELETE_ORIGINAL_FILE,
```

Comment:
```suggestion
  deleteOriginalFile: process.env.DELETE_ORIGINAL_FILE === "true",
```

---

Environment variables comes through as a string, so this will be either "true" or "false", which means the `if` check in `index.ts` will always be truthy and delete the file:

```ts
if (config.deleteOriginalFile) {
```